Plugins/UnityClientPlugin/WebRtcWrapper/WebRtcControl.cs

```diff
@@ -57,15 +57,16 @@ public void Initialize()
             Microphones = new ObservableCollection<MediaDevice>();
             AudioPlayoutDevices = new ObservableCollection<MediaDevice>();
 
-            foreach (MediaDevice audioCaptureDevice in Conductor.Instance.Media.GetAudioCaptureDevices())
-            {
-                Microphones.Add(audioCaptureDevice);
-            }
+            // WebRTCUWP M58 library does not support audio capture/playout devices

```

Comment:
@anderm do we want this?

Reply:
@bengreenier There is no way to access Audio Capture or Output devices with the M58 UWP library. We will update the code as soon as a fix is pushed.